src/ontology/mondo-edit.obo

```diff
@@ -183393,6 +183395,7 @@ id: MONDO:0009667
 name: muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type A3
 def: "An autosomal recessive muscular dystrophy caused by mutations in the POMGNT1 gene. It is associated with characteristic brain and eye malformations, profound mental retardation, and death usually in the first years of life." [NCIT:C126740]
 synonym: "MDDGA3" RELATED ABBREVIATION [MONDO:Lexical, OMIM:253280]
+synonym: "muscle-eye-brain-POMGNT1 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/